src/ripple/conditions/impl/utils.h

```diff
@@ -175,19 +172,20 @@ parsePreamble(Slice& s, std::error_code& ec)
 
         if (p.length == 0)
         {
-            ec = error::generic;
+            ec = error::malformed_encoding;
             return p;
         }
     }
 
-    ec = {};
+
     return p;
 }
 
 inline
 Buffer
 parseOctetString(Slice& s, std::uint32_t count, std::error_code& ec)
 {
+    // We may want to set a max content size. Do we really want to allow 4 gig messages?
```

Comment:
I have no idea what would be a good max. I have a hard time imagining an octet string of conditions being greater than a couple of megs.